src/components/home.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import'./index.css';
+
+
+export const Home = (props) =>{
+  return (
+    <div className = "text">
+    <h1>Pick A List</h1>
+    <div>
+    <MainArticle 
+    title = "Categories"
+    />
+    </div>
+    <div className="home">
+    <Article  
+    Img = "https://res.cloudinary.com/cn-project/image/upload/v1641481525/pana/Winter_solstice-pana_x4wbwx.png" 
```

Comment:
can we change it to something lower case? "image" if we don't want to use "img"

Reply:
We can but it works as it is and it helps differentiate between the main pages images in css